src/components/Projects.tsx

```diff
@@ -37,18 +51,17 @@ export default function Projects() {
             <div id="Projects" className="border-light-brown bg-dark text-white px-20 pt-24 lg:px-40">
             <div className="sm:flex sm:justify-between sm:items-baseline">
                 <div className="sm:w-0 sm:flex-1">
-                <h1 id="message-heading" className="text-2xl font-medium">
-                    Full-Stack <span className="text-light-brown">Developer</span>
+                <h1 id="message-heading" className="text-3xl font-medium">
+                    Personal <span className="text-light-brown">Portfolio</span>
                 </h1>
-                <p className="mt-1 text-sm text-gray-500 truncate">Checkout and Payments Team</p>
+                <p className="mt-1 text-lg text-gray-500 truncate">You can check some of my Github project repositories below:</p>
                 </div>
             </div>
             </div>
 
             <div className="grid grid-cols-1 gap-4 md:grid-cols-2 lg:px-60 bg-dark text-white px-12 pb-6">
                 {projects.map((project) => (
                 <div
```

Comment:
Why you remove the key here??

Reply:
As the email is the same for all items, I thought that is was not necessary. I added it again.